taipy/core/_orchestrator/_dispatcher/_standalone_job_dispatcher.py

```diff
@@ -52,3 +52,7 @@ def _release_worker(self, _):
     def _update_job_status_from_future(self, job: Job, ft):
         self._pop_dispatched_process(job.id)  # type: ignore
         self._update_job_status(job, ft.result())
+
+    def stop(self):
+        super().stop()
+        self._executor.shutdown(wait=True, cancel_futures=True)
```

Comment:
```suggestion
        self._executor.shutdown(wait=True)
```

Python 3.8 does not have the `cancel_futures` parameter. I tried to remove it and it's still working consistently.

Reply:
Did a test catch it?

Reply:
yep. Many, actually.

Reply:
Yes a lot of tests are failing.